libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -34,6 +35,7 @@ export class BookSearchComponent {
 
   addBookToReadingList(book: Book) {
     this.store.dispatch(addToReadingList({ book }));
+    this.store.dispatch(undoAddToReadingList({ book }));
```

Comment:
The triggering of the Snackbar should not happen here. Think of this scenario. You click "add to reading list" and then there's an error and your book is not added to reading list. Then you will have the ability to "undo" an action that has not been confirmed. And this cannot happen! :)
So, it would be best to trigger the snackbar after the add to reading list action has been confirmed!

Reply:
@mandarini Sure I can remove it from here. I was wondering if it is a good practice to dispatch multiple actions in one effect. 
I can dispatch one more action,  ReadingListActions.undoAddToReadingList in the addBook$ effect. 

Please suggest if this a good practice.

Reply:
you should dispatch the undo action after the confirmAddToReadingList action. You can do this in effects, sure